TASK 1: The Hypotheses

Null Hypothesis: Statement: The weekly operating cost is equal to the theoretical cost model. ($\mu = 4000$).
alternate Hypothesis: Statement: The weekly operating cost is higher than the theoretical cost model. ($\mu > 4000$).

In [1]:
sample_mean = 3050     
n = 25                 
units_mean = 600      
fixed_cost = 1000     
variable_cost = 5      
theoretical_mean = fixed_cost + (variable_cost * units_mean)
std_dev_units = 25
std_dev_cost = variable_cost * std_dev_units 
print(f"Sample Mean (x_bar): {sample_mean}")
print(f"Theoretical Mean Cost (Mu): {theoretical_mean}")
print(f"Standard Deviation of Cost (Sigma): {std_dev_cost}")

Sample Mean (x_bar): 3050
Theoretical Mean Cost (Mu): 4000
Standard Deviation of Cost (Sigma): 125


TASK 2: Calculate the Test Statistic (t-score)

In [2]:
import numpy as np
numerator = sample_mean - theoretical_mean
denominator = std_dev_cost / np.sqrt(n)
test_statistic = numerator / denominator

print(f"Numerator: {numerator}")
print(f"Denominator: {denominator}")
print(f"Test Statistic (t): {test_statistic}")

Numerator: -950
Denominator: 25.0
Test Statistic (t): -38.0


In [4]:
import scipy.stats as stats
alpha = 0.05
critical_value = stats.norm.ppf(1 - alpha)
print(f"Alpha level: {alpha}")
print(f"Critical Value : {critical_value}")

Alpha level: 0.05
Critical Value : 1.6448536269514722


In [5]:
print(f"Test Statistic: {test_statistic}")
print(f"Critical Value: {critical_value}")
if test_statistic > critical_value:
    decision = "Reject the Null Hypothesis (H0)."
    evidence = "Strong evidence."
else:
    decision = "Fail to Reject the Null Hypothesis (H0)."
    evidence = "Not enough evidence."
print(decision)
print("\n--- CONCLUSION ---")
print(f"Based on the decision, there is {evidence} to support the restaurant owners' claim.")
print("The data shows the actual cost (3050) is significantly LOWER than the model cost (4000), not higher.")

Test Statistic: -38.0
Critical Value: 1.6448536269514722
Fail to Reject the Null Hypothesis (H0).

--- CONCLUSION ---
Based on the decision, there is Not enough evidence. to support the restaurant owners' claim.
The data shows the actual cost (3050) is significantly LOWER than the model cost (4000), not higher.


# Final Report & Conclusion

### 1. Statistical Summary
* **Test Statistic (t-score):** -38.0
* **Critical Value (Z-value):** +1.645
* **Significance Level ($\alpha$):** 0.05 (5%)

### 2. Decision Logic
This was a right-tailed test because the claim was that the operating costs are *higher*. To support the owners' claim, the test statistic needed to be greater than the critical value (Test Statistic > 1.645). 
However, our test statistic was **-38.0**, which does not fall in the rejection region. Therefore, we **Fail to Reject the Null Hypothesis ($H_0$)**.

### 3. Business Conclusion
There is **no statistical evidence** to support the restaurant owners' claim that the weekly operating costs are higher than the company's theoretical model. In fact, the data heavily indicates the complete opposite: the actual average cost (Rs. 3050) is significantly *lower* than the model's predicted cost (Rs. 4000). The owners' claim is invalid.